sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -1491,7 +1491,14 @@ public virtual async Task<Response<RecognizeLinkedEntitiesResultCollection>> Rec
                 {
                     case 200:
                         IDictionary<string, int> map = CreateIdToIndexMap(documents);
-                        return await CreateLinkedEntityResponseAsync(response, map, cancellationToken).ConfigureAwait(false);
+                        RecognizeLinkedEntitiesResultCollection results = await CreateLinkedEntityResponseAsync(response, map, cancellationToken).ConfigureAwait(false);
```

Comment:
Is there any need to have this on other client methods as well?

Reply:
yes, although currently only linked entities has it. 
I will add the logic to the rest of the endpoints